pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -122,7 +122,7 @@
     @Path("/{property}/{cluster}/{namespace}/destinations")
     @ApiOperation(hidden = true, value = "Get the list of all the topics under a certain namespace.",
             response = String.class, responseContainer = "Set")
-    @ApiResponses(value = {@ApiResponse(code = 403, message = "Don't have admin permission"),
+    @ApiResponses(value = {@ApiResponse(code = 403, message = "Don't have admin or consume permission on namespace"),
```

Comment:
Maybe we need to modify the message, such as "Don't have admin or operate policies permission on the namespace".

Reply:
@gaoran10 good point. The risk here is that custom implementations won't necessarily align with needing admin privileges. I think making it an "or" like you proposed sounds good to me.

Reply:
So “ Don't have operate policies permission on the namespace” is better? @michaeljmarshall @gaoran10

Reply:
@gaoran10 @michaeljmarshall Thanks for you suggestion :)
This is a correction of known inaccurate annonations, not limited to `namespace polocies` operation, including: `getTopics`, `clearNamespaceBundleBacklog`, `unsubscribeNamespace`, etc. so perhaps `Don't have admin or operate permission on the namespace` is more general, what do you think?